apps/web/app/(auth)/auth/components/SlackButton.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import { signIn } from "next-auth/react";
+import { FaSlack } from "react-icons/fa";
+
+import { Button } from "@formbricks/ui/Button";
+
+interface SlackButtonProps {
+  environmentId: string;
+  text?: string;
+  inviteUrl?: string | null;
+}
+
+export const SlackButton = ({ environmentId, text = "Continue with Slack" }: SlackButtonProps) => {
+  const handleLogin = async () => {
+    await signIn("slack", {
+      redirect: true,
+      callbackUrl: `/api/slack/callback?environment=${environmentId}`, // redirect after login to /
+    });
+  };
```

Comment:
Can you please debug and fix the `callbackUrl`, in case of canceling the slack integration, app throws the error.
![image](https://github.com/formbricks/formbricks/assets/56182734/f1821e9e-e241-4a7c-ae2d-3b18edc1b4db)